### ISSUE TITLE
support es6 wepack config 

### ISSUE THREAD

**Issue by lionelB** [Yomguithereal/kotatsu#89]
it seems that webpack config file written in es6 (using `import`, `const` and `=>`)

```
Error: Error while loading your config file: "webpack.config.js".
SyntaxError: Unexpected reserved word
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:414:25)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /Users/lionel/Code/lumographe.fr/node_modules/kotatsu/cli.js:65:25
    at /Users/lionel/Code/lumographe.fr/node_modules/kotatsu/node_modules/yargs/lib/validation.js:169:22
    at Array.forEach (native)
```

**Comment by Yomguithereal** [comment]
Hello @lionelB. Does webpack even works when you feed it an ES6 config?

**Comment by lionelB** [comment]
I'm on it to refine the bug.
Actually, naming a file `webpack.config.babel.js` does the trick in webpack land. 
https://github.com/webpack/webpack/issues/1403

**Comment by Yomguithereal** [comment]
I was not aware of `webpack.config.babel.js`. Seems like a nice trick. Does this work with kotatsu? I guess not.

**Comment by Yomguithereal** [comment]
Webpack seems to rely on [interpret](https://github.com/js-cli/js-interpret) to perform this precise job. I can therefore try something if you want.

**Comment by Bnaya** [comment]
I had working webpack.config.babel.js
I switched from babel 5 to 6 in my project, reinstalled the node modules and now:
```
{ import webpack f
                                                              ^^^^^^
SyntaxError: Unexpected reserved word
```

**Comment by Yomguithereal** [comment]
Released `0.9.0` with ES6 config support.

**Comment by lionelB** [comment]
GG! And thanks
